certificate-analytics-service/app/src/main/java/se/inera/intyg/certificateanalyticsservice/application/messages/model/v1/CertificateAnalyticsEventMessageV1.java

```diff
@@ -19,6 +19,7 @@ public class CertificateAnalyticsEventMessageV1 implements CertificateAnalyticsM
   CertificateAnalyticsEventCertificateV1 certificate;
   CertificateAnalyticsEventV1 event;
   CertificateAnalyticsEventRecipientV1 recipient;
+  CertificateAnalyticsAdministrativeMessageV1 administrativeMessage;
```

Comment:
Not sure about the name but if we only name it message it gets confusing to message for the event

Reply:
I agree it gets confusing. I do think it is better to make some adjustments on naming of the other classes so we can call the attribute message instead. The message parts can be both administrative and complement message even though both could be considered "administrative".